src/dmd/clone.d

```diff
@@ -211,7 +211,7 @@ extern (C++) FuncDeclaration buildOpAssign(StructDeclaration sd, Scope* sc)
     //printf("StructDeclaration::buildOpAssign() %s\n", sd.toChars());
     StorageClass stc = STC.safe | STC.nothrow_ | STC.pure_ | STC.nogc;
     Loc declLoc = sd.loc;
-    Loc loc = Loc(); // internal code should have no loc to prevent coverage
+    Loc loc; // internal code should have no loc to prevent coverage
```

Comment:
Can't this be removed and replace usages of this with `Loc.initial`?

Reply:
That was my first thought. But `Loc.initial` is a global, and `loc` can be 0-initialized if left alone, so I thought default initializing it would be better.